tests/backup/backup_longevity_test.go

```diff
@@ -0,0 +1,1567 @@
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/portworx/torpedo/pkg/log"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	k8s "github.com/portworx/torpedo/drivers/scheduler/k8s"
+	. "github.com/portworx/torpedo/tests"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	testTriggersConfigMap = "longevity-triggers"
+	configMapNS           = "default"
+	controlLoopSleepTime  = time.Second * 15
+)
+
+var (
+	// Stores mapping between chaos level and its freq. Values are hardcoded
+	triggerInterval map[string]map[int]time.Duration
+	// Stores which are disruptive triggers. When disruptive triggers are happening in test,
+	// other triggers are allowed to happen only after existing triggers are complete.
+	disruptiveTriggers map[string]bool
+
+	triggerFunctions     map[string]func(*[]*scheduler.Context, *chan *EventRecord)
+	emailTriggerFunction map[string]func()
+
+	// Pure Topology is disabled by default
+	pureTopologyEnabled = false
+
+	//Default is allow deploying apps both in storage and storageless nodes
+	hyperConvergedTypeEnabled = true
+
+	// Pure Topology Label array
+	labels []map[string]string
+)
+
+// TriggerFunction represents function signature of a testTrigger
+type TriggerFunction func(*[]*scheduler.Context, *chan *EventRecord)
+
+var _ = Describe("{BackupLongevity}", func() {
+	contexts := make([]*scheduler.Context, 0)
+	var triggerLock sync.Mutex
+	// var emailTriggerLock sync.Mutex
+	var populateDone bool
+	triggerEventsChan := make(chan *EventRecord, 100)
+	triggerFunctions = map[string]func(*[]*scheduler.Context, *chan *EventRecord){
+		CreatePxBackup: TriggerCreateBackup,
+	}
+	//Creating a distinct trigger to make sure email triggers at regular intervals
+	emailTriggerFunction = map[string]func(){
+		EmailReporter: TriggerEmailReporter,
+	}
+
+	BeforeEach(func() {
+		if !populateDone {
+			StartPxBackupTorpedoTest("BackupLongevityTest",
```

Comment:
If I recall correctly, adding the tag "longevity": "true" is needed for Aetos to skip performing cleanup.

*BackupLongevity~Test~

Reply:
Done